infrastructure/testdriver/click_iframe.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>TestDriver click method</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+
+<iframe src="click_child.html"></iframe>
+
+<script>
+async_test(t => {
+  addEventListener("load", t.step_func(() => { 
+      let child = frames[0];
+      let button = child.document.getElementById("button");
```

Comment:
Is the iframe definitely loaded at this point?

Reply:
Yes, the load event on the document is blocked on iframes loading.

Reply:
TIL! Thanks :)